docs/contribution-guide.mdx

```diff
@@ -0,0 +1,90 @@
+---
+title: "Contribution guide"
+---
+
+Thank you for interest in contributing to Ship. Here, you'll find all the necessary information to get started.
+
+## Ways to contribute
+
+- **Fix Ship docs** – fixing issues like bad wording, incomplete information, or missing examples in the documentation.
+- **Fix Ship template** - contribute to resolving bugs and enhancing the existing code in current template.
+- **Give feedback and suggest improvements** - report bugs through [issues](https://github.com/paralect/ship/issues) or suggest improvements.
+- **Share Ship** - share Ship link with everyone how can be interested.
+
+## How to open pull request
+1. Fork the [project](https://github.com/paralect/ship).
+2. Set up the project locally.
+3. Create a new branch from the `master` branch, using a name that describes your changes.
+4. Implement your changes and create a commit.
+5. Open a pull request.
+
```

Comment:
also add step, when you need to attach a label (To Review + label with functionality type: improveemnt, bug, feature.. and where you edited it: template, create-ship-app, docs and etc)